packages/next-plugin-material-ui/src/get-styles-server.js

```diff
@@ -0,0 +1,3 @@
+export default async function getStyles (ctx) {
+  return ctx.sheets.getStyleElement()
```

Comment:
This feels odd to has as a separate hook, but should probably be unstable too -- in a streaming world this old method of CSS collection doesn't work.

Reply:
Updated to `unstable`